pages/tools/sdk.md

```diff
@@ -36,6 +37,9 @@ If you feel something is missing, not clear or could be improved, please don't h
             * [.move(uuid, application)](#resin.models.device.move) ⇒ <code>Promise</code>
             * [.restart(uuid)](#resin.models.device.restart) ⇒ <code>Promise</code>
             * [.reboot(uuid)](#resin.models.device.reboot) ⇒ <code>Promise</code>
+            * [.shutdown(uuid)](#resin.models.device.shutdown) ⇒ <code>Promise</code>
```

Comment:
Were these changes made directly on sdk.md? i think these are pulled in dynamically no?